src/components/Map/Map.jsx

```diff
@@ -8,13 +8,24 @@ import MarkerIcon from '../../assets/MarkerIcon.svg';
 import './Map.css';
 import { FYABackend } from '../../common/utils';
 
+class BoxProvider extends OpenStreetMapProvider {
+  constructor(options) {
+    super({
+      ...options,
+      searchUrl: 'http://localhost:3001/anchorBox/search',
```

Comment:
We might want to use environmental variables here since in production the URL will be different!